src/components/SwitchTheme/SwitchTheme.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconicButton } from 'src/components/_common/IconicButton/IconicButton';
+import { SwitchThemeIcon } from 'src/components/_common/icons/SwitchThemeIcon';
+
+import { useSwitchTheme } from './hook';
+
+type SwitchProps = {
+  showIcon?: boolean;
```

Comment:
Last time we decided to have here non-nullable type.

Reply:
okay